### PR TITLE
Add ParallelCluster CPU size 32 results, updaetd READMEs, fixed scripts

### PR DESCRIPTION
**Pull request by milroy**
This PR adds the complete results for ParallelCluster CPU size 32 along with fixed and updated READMEs and scripts.

### CODE REVIEW

File: experiments/aws/eks/gpu/eks-config.yaml [hunk 1]
```diff
@@ -35,4 +35,21 @@ managedNodeGroups:
...
+    ImdsSupport: v2.0
+DevSettings:
+  Cookbook:
+    # https://github.com/aws/aws-parallelcluster/issues/6230
+    # https://github.com/aws/aws-parallelcluster-cookbook/pull/2615
+    # This shows also for OpenRM
+    # https://www.nvidia.com/download/driverResults.aspx/203692/en-us/
+    # 525.105.17
+    ExtraChefAttributes: |
+      {"cluster": {"nvidia": {"enabled": true, "driver_version": "535.161.08", "kernel_open": "false" }}}
```
Comment (vsoch) [inline experiments/aws/eks/gpu/eks-config.yaml:55]:
```suggestion
```

File: experiments/aws/parallel-cluster/cpu/experiment/configs/lammps/slurm-lammps-32n.sh [hunk 2]
```diff
@@ -7,8 +7,7 @@
 
 echo "Start time:" $( date +%s )
 /usr/bin/time -p mpirun -N 2 --map-by ppr:96:node \
```
Comment (vsoch) [inline experiments/aws/parallel-cluster/cpu/experiment/configs/lammps/slurm-lammps-32n.sh:9]:
```suggestion
/usr/bin/time -p mpirun -N 32 --map-by ppr:96:node \
```

File: experiments/azure/cyclecloud/gpu/README.md [hunk 3]
```diff
@@ -94,7 +94,11 @@ oras login ghcr.io --username vsoch
 srun --time=00:04 -N 2 slurm-single-node-benchmarks.sh
 rm -rf test_file*
 cd ../../data/single-node-benchmarks
+<<<<<<< HEAD
```
Comment (vsoch) [inline experiments/azure/cyclecloud/gpu/README.md:97]: Looks like a git rebase leftover few lines here - the top section is correct.

File: experiments/azure/cyclecloud/gpu/README.md [hunk 4]
```diff
@@ -106,7 +110,11 @@ All the following examples are for 32 nodes. Mutatis mutandis for other sizes.
 cd configs/amg2023/
 for i in {1..5}; do sbatch --output=../../data/amg2023/%x-%j-iter-${i}.out --error=../../data/amg2023/%x-%j-iter-${i}.err slurm-amg-32n.sh; done
 cd ../../data/amg2023
+<<<<<<< HEAD
```
Comment (vsoch) [inline experiments/azure/cyclecloud/gpu/README.md:113]: Here as well! And several more in this file (read down).

File: experiments/aws/parallel-cluster/cpu/experiment/results/32-node/stream/stream-1n-226-iter-1.err [hunk 5]
```diff
@@ -0,0 +1,3 @@
+real 0.68
```
Comment (vsoch) [inline experiments/aws/parallel-cluster/cpu/experiment/results/32-node/stream/stream-1n-226-iter-1.err:1]: Just to double check - this file pattern of error (.err) corresponds to the output file (.out) with a slightly different prefix (queue1-dy-*)?

File: experiments/aws/parallel-cluster/cpu/experiment/results/32-node/amg2023/amg-32n-20-iter-1.err [hunk 6]
```diff
@@ -0,0 +1,20 @@
+--------------------------------------------------------------------------
```
Comment (vsoch) [inline experiments/aws/parallel-cluster/cpu/experiment/results/32-node/amg2023/amg-32n-20-iter-1.err:1]: Note for all runs - you can delete the `.gitkeep`s if you like! But no harm to have.

File: experiments/aws/eks/gpu/eks-config.yaml (hunk 1 above)
Comment (milroy) [reply experiments/aws/eks/gpu/eks-config.yaml:55]: AHH, it's in there again!?

**Review by vsoch — APPROVED** [review]

**Pull request merged**